src/components/Expander/Expander.jsx

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
 import React from 'react';
-import PropTypes from 'react-peek/prop-types';
+import PropTypes from 'prop-types';
```

Comment:
did it not work with peek?   We'll have to figure out what sequence of refactor tasks allow docs to work during every step.

Reply:
Nope, didn't work with peek